WebService/appsettings.ini

```diff
@@ -31,6 +31,12 @@ webservice_url = "${PCS_STORAGEADAPTER_WEBSERVICE_URL}"
 # Max time allowed for requests to the storage adapter before reporting an error
 webservice_timeout = 10000
 
+[AzureManagementService]
+# URL where the azure management api is available, e.g. http://localhost:9022/v1 on local dev environments
+webservice_url = "https://management.azure.com"
+
+# Max time allowed for requests to the azure management adapter before reporting an error
```

Comment:
replace adapter with api. "ax time allowed for requests to the azure management api before reporting an error"